templateMaker/src/getZmassV2.cpp

```diff
@@ -0,0 +1,62 @@
+#include "getZmassV2.hpp"
```

Comment:
time to rename this class! what about Zselection?
ps Defines here can be made using RVec syntax
check /scratchnvme/emanca/wproperties-analysis/templateMaker/src/Zselection.cpp